site/content/docs/concepts/pipelines.md

```diff
@@ -24,14 +24,14 @@ Want to learn more about CodePipeline? Check out their [getting started docs](ht
 Creating a Pipeline requires only three steps:
 
 1. Preparing the pipeline structure.
-2. Committing the generated `buildspec.yml`.
+2. Committing and pushing the generated `pipeline.yml` and `buildspec.yml`.
```

Comment:
The `.workspace` file needs to also be there to pick up the app name. So anything under `copilot/`